src/drivers/uavcan/uavcan_params.c

```diff
@@ -341,7 +365,7 @@ PARAM_DEFINE_INT32(UAVCAN_SUB_IMU, 0);
  * @reboot_required true
  * @group UAVCAN
  */
-PARAM_DEFINE_INT32(UAVCAN_SUB_MAG, 1);
+PARAM_DEFINE_INT32(UAVCAN_SUB_MAG, 0);
```

Comment:
The same as above

Reply:
yep, I moved it there https://github.com/tiiuae/saluki-v1/pull/69